apps/client/src/common/stores/clientStore.ts

```diff
@@ -0,0 +1,62 @@
+import { Clients } from 'ontime-types';
+import { create } from 'zustand';
+
+interface ClientStore {
+  name?: string;
+  setMyName: (newValue: string) => void;
```

Comment:
suggestion: the function name should be connected to the property it affects on. So for a property `name` we would prefer having a `setName` function